pmemcheck/tests/common.h

```diff
@@ -12,20 +12,14 @@
  * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
  * more details.
  */
-#include <stdio.h>
+
 #include <stdlib.h>
-#include <string.h>
-#include <sys/types.h>
-#include <sys/param.h>
-#include <signal.h>
 #include <fcntl.h>
 #include <unistd.h>
 #include <errno.h>
-#include <stdint.h>
 #include <sys/mman.h>
-#include <sys/stat.h>
 
-#include "pmemcheck/pmemcheck.h"
+#include "../pmemcheck.h"
```

Comment:
Don't like this style of including. What about adding `-I..` to the compiler options.